ksql-rest-app/src/main/java/io/confluent/ksql/rest/entity/EntityQueryId.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.ksql.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import io.confluent.ksql.query.QueryId;
+
+import java.util.Objects;
+
+public class EntityQueryId {
```

Comment:
Why not just `QueryId`?

Reply:
As discussed - the intent here is to keep the api types decoupled from internal types since its important that we not make breaking changes to the api.